caiman/utils/utils.py

```diff
@@ -82,7 +82,15 @@ def download_demo(name='Sue_2x_3000_40_-46.tif', save_folder=''):
             url = file_dict[name]
             logging.info(f"downloading {name} with urllib")
             logging.info(f"GET {url} HTTP/1.1")
-            f = urlopen(url)
+            try:
+                f = urlopen(url)
+            except:
+                logging.info(f"Trying to set user agent to download demo")
```

Comment:
@agiovann typo here?

Reply:
Is bad english the problem?

Reply:
There is an "f" before the quotes and the tests are failing

Reply:
I coped from the lines 83-84 above the syntax. Not sure why it is failing... works for me

Reply:
The f should not be an issue - that's a standard python feature called an f-string.